packages/kbn-discover-utils/src/__mocks__/es_hits.ts

```diff
@@ -60,7 +60,7 @@ export const esHitsMockWithSort = esHitsMock.map((hit) => ({
 }));
 
 const baseDate = new Date('2024-01-1').getTime();
-const dateInc = 100_000_000;
+const dateInc = 100000000;
```

Comment:
Storybook failed because of this

Reply:
Well that's strange since it should just be syntactic sugar 🤔 any idea why this caused failures?

Reply:
I've revisited this, and reverted this change ... now it works?! Might be that the bug was me? Anyway. thx for asking, 1 redundant change less 👍